### PR TITLE
New Router Label

### PR DESCRIPTION
**Pull request by brianteeman**
PR for #17673

This didnt make it into 3.8.0 as we were in language freeze.

This PR renames the router options as they were originaly intended to be to Modern and Legacy from Experimental and Stable

### CODE REVIEW

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on 79d37d78ec32597ff4923f8520ceee162aa650da<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/18165">issues.joomla.org/tracker/joomla-cms/18165</a>.</sub>

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on 79d37d78ec32597ff4923f8520ceee162aa650da<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/18165">issues.joomla.org/tracker/joomla-cms/18165</a>.</sub>

**Comment by ghost** [comment]
RTC after two successful tests.

File: administrator/language/en-GB/en-GB.ini [hunk 1]
```diff
@@ -511,10 +511,10 @@ JGLOBAL_SAVE_HISTORY_OPTIONS_DESC="Automatically save old versions of an item. I
 JGLOBAL_SAVE_HISTORY_OPTIONS_LABEL="Enable Versions"
 JGLOBAL_SECRETKEY="Secret Key"
 JGLOBAL_SECRETKEY_HELP="If you have enabled two factor authentication in your user account please enter your secret key. If you do not know what this means, you can leave this field blank."
-JGLOBAL_SEF_ADVANCED_DESC="Experimental Routing enables advanced features but may change your URLs. Stable routing ensures full compatibility for existing sites. This is configured per component."
+JGLOBAL_SEF_ADVANCED_DESC="Modern Routing enables advanced features but may change your URLs. Legacy routing ensures full compatibility for existing sites. This is configured per component."
```
Comment (Quy) [inline administrator/language/en-GB/en-GB.ini]: Lower case `R` in `Routing`?

Comment (brianteeman) [reply administrator/language/en-GB/en-GB.ini]: thanks - it was an error in the original string

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**